docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -165,21 +198,39 @@ insert into users (email, name, state) values ('sue@help.com', 'Sue Sas', 'CA');
 insert into users (email, name, state) values ('tom@yes.com', 'Tom and Jerry', 'NV');
 ```
 
-Copy the cql file to the k8ssandra container (pod) :
+Copy the cql file to the k8ssandra container (pod):
 
-`kubectl cp test_data.cql k8ssandra-dc1-default-sts-0:/tmp -c cassandra`
+`kubectl cp test_data.cql demo-dc1-default-sts-0:/tmp -c cassandra`
 
-Add the data to the Cassandra database:
+Before you can launch the CQLSH instance that's deployed by K8ssandra in your Kubernetes cluster, you'll need authentication credentials. The superuser secret defaults to `<cluster-name>-superuser`. In this example: `demo-superuser`. To verify, you can extract and decode the username secret:
```

Comment:
We should have a section in the docs that describe the steps necessary for accessing C* credentials and accessing cqlsh (not sure if that exists yet). Should we just link to that section here?

Reply:
We'll have a separate topic or a section with the Quick Start that describes accessing C* credentials and cqlsh, yes. But users get lost if we xref them to another topic in the middle of a step-by-step procedure. So I'm inclined to explicitly show the credentials examples here too, even though it duplicates content.